src/component/feature/variant/AddVariant/AddVariant.jsx

```diff
@@ -264,24 +268,36 @@ const AddVariant = ({
                         />
                     </Grid>
                 </Grid>
-                {overrides.length > 0 && (
-                    <p style={{ marginBottom: '.5rem' }}>
-                        <strong>Overrides </strong>
-                        <Icon
-                            name="info"
-                            title="Here you can specify which users that should get this variant."
-                        />
-                    </p>
-                )}
 
-                <OverrideConfig
-                    overrides={overrides}
-                    removeOverride={removeOverride}
-                    updateOverrideType={updateOverrideType}
-                    updateOverrideValues={updateOverrideValues}
-                    updateValues={updateOverrideValues}
+                <ConditionallyRender
+                    condition={uiConfig.flags[C]}
```

Comment:
Sorry for the confusion. We don't need to check this flag as overrides are available for all.